tox.ini

```diff
@@ -17,24 +17,27 @@ setenv =
 [testenv:base]
 description = Tests base modules
 commands =
-    pytest --cov "{envsitepackagesdir}/pydvl" -m "not torch" {posargs}
+    pytest tests/ --cov "{envsitepackagesdir}/pydvl" -m "not torch" {posargs}
```

Comment:
There is no need to specify the path here because it is the default we have [configured](https://github.com/appliedAI-Initiative/pyDVL/blob/64b4ebcf21380c5a5357980bc314171e3fb6a9e8/pyproject.toml#L17)

Reply:
When removing the test directory and run tox locally. i.e. tox -e base, the pytest ini default directory is ignored and pytest pulls test from the notebooks folder as well. This is weird, since in CI it works. Can you run tox -e base locally without the explicit directory?

Reply:
For me it works the same as it does on CI. Which version of tox are you using? Perhaps there's a bug in a newer version.

Could you running this command to see which test files are collected? 

```shell
tox -e base -- --collect-only
```

Reply:
This is the result I get:

```shell
<Package tests>
  <Module test_plugin.py>
    <Function test_marker_only[0]>
    <Function test_marker_only_with_data_fixture[0]>
    <Function test_marker_only_with_data_fixture[1]>
    <Function test_marker_only_with_data_fixture[2]>
    <Function test_fixture_only[1]>
    <Function test_fixture_only[2]>
    <Function test_fixture_call_no_arguments>
    <Function test_marker_and_fixture[1]>
    <Function test_marker_and_fixture[2]>
    <Function test_failure>
    <Function test_marker_ignore_exception[0]>
    <Function test_marker_ignore_exception[1]>
    <Function test_marker_ignore_exception[2]>
    <Function test_marker_ignore_exception[3]>
    <Function test_marker_ignore_exception[4]>
  <Module test_results.py>
    <Function test_sorting[values0-names0-ranks_asc0]>
    <Function test_sorting[values1-names1-ranks_asc1]>
    <Function test_dataframe_sorting[values0-names0-ranks_asc0]>
    <Function test_dataframe_sorting[values1-names1-ranks_asc1]>
    <Function test_iter[values0-names0-ranks_asc0]>
    <Function test_iter[values1-names1-ranks_asc1]>
    <Function test_todataframe[values0-names0-ranks_asc0]>
    <Function test_todataframe[values1-names1-ranks_asc1]>
    <Function test_indexing[values0-names0-ranks_asc0]>
    <Function test_indexing[values1-names1-ranks_asc1]>
    <Function test_get_idx>
    <Function test_updating>
    <Function test_serialization[values0-None-dumps-loads0]>
    <Function test_serialization[values0-None-dumps-loads1]>
    <Function test_serialization[values1-None-dumps-loads0]>
    <Function test_serialization[values1-None-dumps-loads1]>
    <Function test_equality[values0-names0]>
    <Function test_equality[values1-names1]>
    <Function test_extra_values[extra_values0]>
    <Function test_extra_values[extra_values1]>
    <Function test_from_random_creation[None-1]>
    <Function test_from_random_creation[None-10]>
    <Function test_from_random_creation[1.0-1]>
    <Function test_from_random_creation[1.0-10]>
    <Function test_from_random_creation[-1.0-1]>
    <Function test_from_random_creation[-1.0-10]>
    <Function test_from_random_creation_errors>
    <Function test_adding_random>
    <Function test_adding_different_indices[indices_10-names_10-values_10-indices_20-names_20-values_20-expected_indices0-expected_names0-expected_values0]>
    <Function test_adding_different_indices[indices_11-names_11-values_11-indices_21-names_21-values_21-expected_indices1-expected_names1-expected_values1]>
    <Function test_adding_different_indices[indices_12-names_12-values_12-indices_22-names_22-values_22-expected_indices2-expected_names2-expected_values2]>
    <Function test_adding_different_indices[indices_13-names_13-values_13-indices_23-names_23-values_23-expected_indices3-expected_names3-expected_values3]>
    <Function test_types[indices0-int32-data_names0-<U1]>
    <Function test_types[indices1-int64-data_names1-int64]>
    <Function test_types[indices2-int32-data_names2-float64]>
    <Function test_names[data_names0]>
    <Function test_empty[0]>
    <Function test_empty[5]>
    <Function test_empty_deprecation>
<Package utils>
  <Module test_caching.py>
    <Function test_failed_connection>
    <Function test_memcached_single_job>
    <Function test_memcached_parallel_jobs[joblib]>
    <Function test_memcached_parallel_repeated_training[joblib-100-1-10-4]>
    <Function test_memcached_parallel_repeated_training[joblib-100-1-20-10]>
    <Function test_memcached_parallel_repeated_training[joblib-100-2-10-4]>
    <Function test_memcached_parallel_repeated_training[joblib-100-2-20-10]>
    <Function test_memcached_parallel_jobs[ray-local]>
    <Function test_memcached_parallel_repeated_training[ray-local-100-1-10-4]>
    <Function test_memcached_parallel_repeated_training[ray-local-100-1-20-10]>
    <Function test_memcached_parallel_repeated_training[ray-local-100-2-10-4]>
    <Function test_memcached_parallel_repeated_training[ray-local-100-2-20-10]>
    <Function test_memcached_parallel_jobs[ray-external]>
    <Function test_memcached_parallel_repeated_training[ray-external-100-1-10-4]>
    <Function test_memcached_parallel_repeated_training[ray-external-100-1-20-10]>
    <Function test_memcached_parallel_repeated_training[ray-external-100-2-10-4]>
    <Function test_memcached_parallel_repeated_training[ray-external-100-2-20-10]>
    <Function test_memcached_repeated_training>
    <Function test_memcached_faster_with_repeated_training>
  <Module test_dataset.py>
    <Function test_creating_dataset_from_sklearn[0.1]>
    <Function test_creating_dataset_subsclassfrom_sklearn[0.1]>
    <Function test_creating_dataset_from_x_y_arrays[0.1-kwargs0]>
    <Function test_creating_dataset_from_x_y_arrays[0.1-kwargs1]>
    <Function test_creating_grouped_dataset_from_sklearn[0.1]>
    <Function test_creating_grouped_dataset_from_sklearn_failure[0.1]>
    <Function test_creating_grouped_dataset_subsclassfrom_sklearn[0.1]>
    <Function test_creating_grouped_dataset_from_x_y_arrays[0.1-kwargs0]>
    <Function test_creating_grouped_dataset_from_x_y_arrays[0.1-kwargs1]>
    <Function test_creating_grouped_dataset_from_x_y_arrays_failure[0.1]>
    <Function test_creating_dataset_from_sklearn[0.5]>
    <Function test_creating_dataset_subsclassfrom_sklearn[0.5]>
    <Function test_creating_dataset_from_x_y_arrays[0.5-kwargs0]>
    <Function test_creating_dataset_from_x_y_arrays[0.5-kwargs1]>
    <Function test_creating_grouped_dataset_from_sklearn[0.5]>
    <Function test_creating_grouped_dataset_from_sklearn_failure[0.5]>
    <Function test_creating_grouped_dataset_subsclassfrom_sklearn[0.5]>
    <Function test_creating_grouped_dataset_from_x_y_arrays[0.5-kwargs0]>
    <Function test_creating_grouped_dataset_from_x_y_arrays[0.5-kwargs1]>
    <Function test_creating_grouped_dataset_from_x_y_arrays_failure[0.5]>
    <Function test_creating_dataset_from_sklearn[0.8]>
    <Function test_creating_dataset_subsclassfrom_sklearn[0.8]>
    <Function test_creating_dataset_from_x_y_arrays[0.8-kwargs0]>
    <Function test_creating_dataset_from_x_y_arrays[0.8-kwargs1]>
    <Function test_creating_grouped_dataset_from_sklearn[0.8]>
    <Function test_creating_grouped_dataset_from_sklearn_failure[0.8]>
    <Function test_creating_grouped_dataset_subsclassfrom_sklearn[0.8]>
    <Function test_creating_grouped_dataset_from_x_y_arrays[0.8-kwargs0]>
    <Function test_creating_grouped_dataset_from_x_y_arrays[0.8-kwargs1]>
    <Function test_creating_grouped_dataset_from_x_y_arrays_failure[0.8]>
    <Function test_grouped_dataset_results>
  <Module test_numeric.py>
    <Function test_powerset>
    <Function test_random_powerset[1-10]>
    <Function test_random_powerset[10-1024]>
    <Function test_random_powerset[5-128]>
    <Function test_random_powerset[0-1]>
    <Function test_random_subset_of_size[0-0-None]>
    <Function test_random_subset_of_size[0-1-ValueError]>
    <Function test_random_subset_of_size[10-0-None]>
    <Function test_random_subset_of_size[10-3-None]>
    <Function test_random_subset_of_size[1000-40-None]>
    <Function test_random_matrix_with_condition_number[1-2-ValueError]>
    <Function test_random_matrix_with_condition_number[0-2-ValueError]>
    <Function test_random_matrix_with_condition_number[4--2-ValueError]>
    <Function test_random_matrix_with_condition_number[10-1-ValueError]>
    <Function test_random_matrix_with_condition_number[2-10-None]>
    <Function test_random_matrix_with_condition_number[7-23-None]>
    <Function test_random_matrix_with_condition_number[10-2-None]>
    <Function test_running_moments>
  <Module test_parallel.py>
    <Function test_effective_n_jobs[joblib]>
    <Function test_map_reduce_job[joblib-1-list-indices0-expected0]>
    <Function test_map_reduce_job[joblib-1-list-indices1-expected1]>
    <Function test_map_reduce_job[joblib-1-list-indices2-expected2]>
    <Function test_map_reduce_job[joblib-1-range-indices3-expected3]>
    <Function test_map_reduce_job[joblib-1-numpy-indices4-45]>
    <Function test_map_reduce_job[joblib-2-list-indices0-expected0]>
    <Function test_map_reduce_job[joblib-2-list-indices1-expected1]>
    <Function test_map_reduce_job[joblib-2-list-indices2-expected2]>
    <Function test_map_reduce_job[joblib-2-range-indices3-expected3]>
    <Function test_map_reduce_job[joblib-2-numpy-indices4-45]>
    <Function test_map_reduce_job[joblib-4-list-indices0-expected0]>
    <Function test_map_reduce_job[joblib-4-list-indices1-expected1]>
    <Function test_map_reduce_job[joblib-4-list-indices2-expected2]>
    <Function test_map_reduce_job[joblib-4-range-indices3-expected3]>
    <Function test_map_reduce_job[joblib-4-numpy-indices4-45]>
    <Function test_chunkification[joblib-data0-3-expected_chunks0]>
    <Function test_chunkification[joblib-data1-2-expected_chunks1]>
    <Function test_chunkification[joblib-data2-2-expected_chunks2]>
    <Function test_chunkification[joblib-data3-3-expected_chunks3]>
    <Function test_chunkification[joblib-data4-5-expected_chunks4]>
    <Function test_chunkification[joblib-data5-42-expected_chunks5]>
    <Function test_chunkification[joblib-data6-42-expected_chunks6]>
    <Function test_chunkification[joblib-data7-4-expected_chunks7]>
    <Function test_chunkification[joblib-data8-4-expected_chunks8]>
    <Function test_map_reduce_job_partial_map_and_reduce_func[joblib]>
    <Function test_wrap_function[joblib]>
    <Function test_futures_executor_submit[joblib]>
    <Function test_futures_executor_map[joblib]>
    <Function test_futures_executor_map_with_max_workers[joblib]>
    <Function test_effective_n_jobs[ray-local]>
    <Function test_map_reduce_job[ray-local-1-list-indices0-expected0]>
    <Function test_map_reduce_job[ray-local-1-list-indices1-expected1]>
    <Function test_map_reduce_job[ray-local-1-list-indices2-expected2]>
    <Function test_map_reduce_job[ray-local-1-range-indices3-expected3]>
    <Function test_map_reduce_job[ray-local-1-numpy-indices4-45]>
    <Function test_map_reduce_job[ray-local-2-list-indices0-expected0]>
    <Function test_map_reduce_job[ray-local-2-list-indices1-expected1]>
    <Function test_map_reduce_job[ray-local-2-list-indices2-expected2]>
    <Function test_map_reduce_job[ray-local-2-range-indices3-expected3]>
    <Function test_map_reduce_job[ray-local-2-numpy-indices4-45]>
    <Function test_map_reduce_job[ray-local-4-list-indices0-expected0]>
    <Function test_map_reduce_job[ray-local-4-list-indices1-expected1]>
    <Function test_map_reduce_job[ray-local-4-list-indices2-expected2]>
    <Function test_map_reduce_job[ray-local-4-range-indices3-expected3]>
    <Function test_map_reduce_job[ray-local-4-numpy-indices4-45]>
    <Function test_chunkification[ray-local-data0-3-expected_chunks0]>
    <Function test_chunkification[ray-local-data1-2-expected_chunks1]>
    <Function test_chunkification[ray-local-data2-2-expected_chunks2]>
    <Function test_chunkification[ray-local-data3-3-expected_chunks3]>
    <Function test_chunkification[ray-local-data4-5-expected_chunks4]>
    <Function test_chunkification[ray-local-data5-42-expected_chunks5]>
    <Function test_chunkification[ray-local-data6-42-expected_chunks6]>
    <Function test_chunkification[ray-local-data7-4-expected_chunks7]>
    <Function test_chunkification[ray-local-data8-4-expected_chunks8]>
    <Function test_map_reduce_job_partial_map_and_reduce_func[ray-local]>
    <Function test_wrap_function[ray-local]>
    <Function test_futures_executor_submit[ray-local]>
    <Function test_futures_executor_map[ray-local]>
    <Function test_futures_executor_map_with_max_workers[ray-local]>
    <Function test_effective_n_jobs[ray-external]>
    <Function test_map_reduce_job[ray-external-1-list-indices0-expected0]>
    <Function test_map_reduce_job[ray-external-1-list-indices1-expected1]>
    <Function test_map_reduce_job[ray-external-1-list-indices2-expected2]>
    <Function test_map_reduce_job[ray-external-1-range-indices3-expected3]>
    <Function test_map_reduce_job[ray-external-1-numpy-indices4-45]>
    <Function test_map_reduce_job[ray-external-2-list-indices0-expected0]>
    <Function test_map_reduce_job[ray-external-2-list-indices1-expected1]>
    <Function test_map_reduce_job[ray-external-2-list-indices2-expected2]>
    <Function test_map_reduce_job[ray-external-2-range-indices3-expected3]>
    <Function test_map_reduce_job[ray-external-2-numpy-indices4-45]>
    <Function test_map_reduce_job[ray-external-4-list-indices0-expected0]>
    <Function test_map_reduce_job[ray-external-4-list-indices1-expected1]>
    <Function test_map_reduce_job[ray-external-4-list-indices2-expected2]>
    <Function test_map_reduce_job[ray-external-4-range-indices3-expected3]>
    <Function test_map_reduce_job[ray-external-4-numpy-indices4-45]>
    <Function test_chunkification[ray-external-data0-3-expected_chunks0]>
    <Function test_chunkification[ray-external-data1-2-expected_chunks1]>
    <Function test_chunkification[ray-external-data2-2-expected_chunks2]>
    <Function test_chunkification[ray-external-data3-3-expected_chunks3]>
    <Function test_chunkification[ray-external-data4-5-expected_chunks4]>
    <Function test_chunkification[ray-external-data5-42-expected_chunks5]>
    <Function test_chunkification[ray-external-data6-42-expected_chunks6]>
    <Function test_chunkification[ray-external-data7-4-expected_chunks7]>
    <Function test_chunkification[ray-external-data8-4-expected_chunks8]>
    <Function test_map_reduce_job_partial_map_and_reduce_func[ray-external]>
    <Function test_wrap_function[ray-external]>
    <Function test_futures_executor_submit[ray-external]>
    <Function test_futures_executor_map[ray-external]>
    <Function test_futures_executor_map_with_max_workers[ray-external]>
  <Module test_score.py>
    <Function test_scorer>
    <Function test_compose_score>
    <Function test_squashed_r2>
    <Function test_squashed_variance>
  <Module test_status.py>
    <Function test_or_status>
    <Function test_and_status>
    <Function test_not_status>
  <Module test_utility.py>
    <Function test_utility_show_warnings[4-4-False]>
    <Function test_utility_show_warnings[4-4-True]>
    <Function test_data_utility_learning_wrapper[2-2-0-8]>
    <Function test_data_utility_learning_wrapper[10-2-0-8]>
    <Function test_cache[2-0-8]>
    <Function test_different_cache_signature[model_kwargs0-2-0-8]>
    <Function test_different_cache_signature[model_kwargs1-2-0-8]>
<Package value>
  <Module test_sampler.py>
    <Function test_proper[indices0-DeterministicCombinatorialSampler]>
    <Function test_proper[indices0-UniformSampler]>
    <Function test_proper[indices0-PermutationSampler]>
    <Function test_proper[indices0-AntitheticSampler]>
    <Function test_proper[indices0-RandomHierarchicalSampler]>
    <Function test_proper[indices1-DeterministicCombinatorialSampler]>
    <Function test_proper[indices1-UniformSampler]>
    <Function test_proper[indices1-PermutationSampler]>
    <Function test_proper[indices1-AntitheticSampler]>
    <Function test_proper[indices1-RandomHierarchicalSampler]>
    <Function test_chunkify[DeterministicCombinatorialSampler]>
    <Function test_chunkify[UniformSampler]>
    <Function test_chunkify[AntitheticSampler]>
    <Function test_chunkify[RandomHierarchicalSampler]>
    <Function test_chunkify_permutation[DeterministicPermutationSampler]>
    <Function test_chunkify_permutation[PermutationSampler]>
  <Module test_semivalues.py>
    <Function test_shapley[_semivalues-shapley_coefficient-criterion0-DeterministicCombinatorialSampler-5]>
    <Function test_shapley[_semivalues-shapley_coefficient-criterion0-DeterministicPermutationSampler-5]>
    <Function test_shapley[_semivalues-shapley_coefficient-criterion0-UniformSampler-5]>
    <Function test_shapley[_semivalues-shapley_coefficient-criterion0-PermutationSampler-5]>
    <Function test_shapley[_semivalues-shapley_coefficient-criterion0-AntitheticSampler-5]>
    <Function test_shapley[_semivalues-beta_coefficient_w-criterion1-DeterministicCombinatorialSampler-5]>
    <Function test_shapley[_semivalues-beta_coefficient_w-criterion1-DeterministicPermutationSampler-5]>
    <Function test_shapley[_semivalues-beta_coefficient_w-criterion1-UniformSampler-5]>
    <Function test_shapley[_semivalues-beta_coefficient_w-criterion1-PermutationSampler-5]>
    <Function test_shapley[_semivalues-beta_coefficient_w-criterion1-AntitheticSampler-5]>
    <Function test_shapley[semivalues-shapley_coefficient-criterion0-DeterministicCombinatorialSampler-5]>
    <Function test_shapley[semivalues-shapley_coefficient-criterion0-DeterministicPermutationSampler-5]>
    <Function test_shapley[semivalues-shapley_coefficient-criterion0-UniformSampler-5]>
    <Function test_shapley[semivalues-shapley_coefficient-criterion0-PermutationSampler-5]>
    <Function test_shapley[semivalues-shapley_coefficient-criterion0-AntitheticSampler-5]>
    <Function test_shapley[semivalues-beta_coefficient_w-criterion1-DeterministicCombinatorialSampler-5]>
    <Function test_shapley[semivalues-beta_coefficient_w-criterion1-DeterministicPermutationSampler-5]>
    <Function test_shapley[semivalues-beta_coefficient_w-criterion1-UniformSampler-5]>
    <Function test_shapley[semivalues-beta_coefficient_w-criterion1-PermutationSampler-5]>
    <Function test_shapley[semivalues-beta_coefficient_w-criterion1-AntitheticSampler-5]>
    <Function test_banzhaf[_semivalues-DeterministicCombinatorialSampler-5]>
    <Function test_banzhaf[_semivalues-DeterministicPermutationSampler-5]>
    <Function test_banzhaf[_semivalues-UniformSampler-5]>
    <Function test_banzhaf[_semivalues-PermutationSampler-5]>
    <Function test_banzhaf[_semivalues-AntitheticSampler-5]>
    <Function test_banzhaf[semivalues-DeterministicCombinatorialSampler-5]>
    <Function test_banzhaf[semivalues-DeterministicPermutationSampler-5]>
    <Function test_banzhaf[semivalues-UniformSampler-5]>
    <Function test_banzhaf[semivalues-PermutationSampler-5]>
    <Function test_banzhaf[semivalues-AntitheticSampler-5]>
    <Function test_coefficients[beta_coefficient_w0-10]>
    <Function test_coefficients[beta_coefficient_w0-100]>
    <Function test_coefficients[beta_coefficient_w1-10]>
    <Function test_coefficients[beta_coefficient_w1-100]>
    <Function test_coefficients[beta_coefficient_w2-10]>
    <Function test_coefficients[beta_coefficient_w2-100]>
    <Function test_coefficients[banzhaf_coefficient-10]>
    <Function test_coefficients[banzhaf_coefficient-100]>
    <Function test_coefficients[shapley_coefficient-10]>
    <Function test_coefficients[shapley_coefficient-100]>
    <Function test_dispatch_compute_semi_values[shapley-5]>
    <Function test_dispatch_compute_semi_values[beta-shapley-5]>
    <Function test_dispatch_compute_semi_values[banzhaf-5]>
  <Module test_stopping.py>
    <Function test_stopping_criterion>
    <Function test_stopping_criterion_composition>
    <Function test_make_criterion>
    <Function test_minmax_updates>
    <Function test_max_time>
    <Function test_history_deviation[0.01-1]>
    <Function test_history_deviation[0.01-42]>
    <Function test_history_deviation[0.01-100]>
    <Function test_history_deviation[0.05-1]>
    <Function test_history_deviation[0.05-42]>
    <Function test_history_deviation[0.05-100]>
    <Function test_standard_error>
    <Function test_max_checks>
<Package least_core>
  <Module test_common.py>
    <Function test_lc_solve_problems[joblib-test_utility0]>
    <Function test_lc_solve_problems[ray-local-test_utility0]>
    <Function test_lc_solve_problems[ray-external-test_utility0]>
  <Module test_montecarlo.py>
    <Function test_montecarlo_least_core[True-1-test_utility0-0.1-128]>
    <Function test_montecarlo_least_core[True-1-test_utility1-0.2-10000]>
    <Function test_montecarlo_least_core[True--1-test_utility0-0.1-128]>
    <Function test_montecarlo_least_core[True--1-test_utility1-0.2-10000]>
    <Function test_montecarlo_least_core[False-1-test_utility0-0.1-128]>
    <Function test_montecarlo_least_core[False-1-test_utility1-0.2-10000]>
    <Function test_montecarlo_least_core[False--1-test_utility0-0.1-128]>
    <Function test_montecarlo_least_core[False--1-test_utility1-0.2-10000]>
  <Module test_naive.py>
    <Function test_naive_least_core[True-test_utility0]>
    <Function test_naive_least_core[False-test_utility0]>
    <Function test_naive_least_core[True-test_utility1]>
    <Function test_naive_least_core[False-test_utility1]>
    <Function test_naive_least_core[True-test_utility2]>
    <Function test_naive_least_core[False-test_utility2]>
    <Function test_naive_least_core[True-test_utility3]>
    <Function test_naive_least_core[False-test_utility3]>
    <Function test_naive_least_core[True-test_utility4]>
    <Function test_naive_least_core[False-test_utility4]>
<Package loo>
  <Module test_loo.py>
    <Function test_naive_loo[10]>
    <Function test_naive_loo[100]>
<Package shapley>
  <Module test_knn.py>
    <Function test_knn_montecarlo_match>
  <Module test_montecarlo.py>
    <Function test_analytic_montecarlo_shapley[joblib-12-permutation_montecarlo-0.1-1e-05-kwargs0]>
    <Function test_analytic_montecarlo_shapley[joblib-8-combinatorial_montecarlo-0.2-0.0001-kwargs1]>
    <Function test_analytic_montecarlo_shapley[joblib-12-owen-0.1-0.0001-kwargs2]>
    <Function test_analytic_montecarlo_shapley[joblib-12-owen_antithetic-0.1-0.0001-kwargs3]>
    <Function test_analytic_montecarlo_shapley[joblib-4-group_testing-0.1-0.01-kwargs4]>
    <Function test_analytic_montecarlo_shapley[ray-local-12-permutation_montecarlo-0.1-1e-05-kwargs0]>
    <Function test_analytic_montecarlo_shapley[ray-local-8-combinatorial_montecarlo-0.2-0.0001-kwargs1]>
    <Function test_analytic_montecarlo_shapley[ray-local-12-owen-0.1-0.0001-kwargs2]>
    <Function test_analytic_montecarlo_shapley[ray-local-12-owen_antithetic-0.1-0.0001-kwargs3]>
    <Function test_analytic_montecarlo_shapley[ray-local-4-group_testing-0.1-0.01-kwargs4]>
    <Function test_analytic_montecarlo_shapley[ray-external-12-permutation_montecarlo-0.1-1e-05-kwargs0]>
    <Function test_analytic_montecarlo_shapley[ray-external-8-combinatorial_montecarlo-0.2-0.0001-kwargs1]>
    <Function test_analytic_montecarlo_shapley[ray-external-12-owen-0.1-0.0001-kwargs2]>
    <Function test_analytic_montecarlo_shapley[ray-external-12-owen_antithetic-0.1-0.0001-kwargs3]>
    <Function test_analytic_montecarlo_shapley[ray-external-4-group_testing-0.1-0.01-kwargs4]>
    <Function test_hoeffding_bound_montecarlo[permutation_montecarlo-8-0.1-0.1]>
    <Function test_hoeffding_bound_montecarlo[combinatorial_montecarlo-8-0.1-0.1]>
    <Function test_linear_montecarlo_shapley[permutation_montecarlo-kwargs0-scorer0-0.25-2-0-21]>
    <Function test_linear_montecarlo_shapley[combinatorial_montecarlo-kwargs1-scorer0-0.25-2-0-21]>
    <Function test_linear_montecarlo_shapley[owen-kwargs2-scorer0-0.25-2-0-21]>
    <Function test_linear_montecarlo_shapley[owen_antithetic-kwargs3-scorer0-0.25-2-0-21]>
    <Function test_linear_montecarlo_shapley[group_testing-kwargs4-scorer0-0.25-2-0-21]>
    <Function test_linear_montecarlo_with_outlier[permutation_montecarlo-kwargs0-scorer0-0.2-2-0-21]>
    <Function test_linear_montecarlo_with_outlier[owen-kwargs1-scorer0-0.2-2-0-21]>
    <Function test_linear_montecarlo_with_outlier[owen_antithetic-kwargs2-scorer0-0.2-2-0-21]>
    <Function test_linear_montecarlo_with_outlier[group_testing-kwargs3-scorer0-0.2-2-0-21]>
    <Function test_grouped_linear_montecarlo_shapley[permutation_montecarlo-kwargs0-scorer0-0.1-2-0-21-2]>
  <Module test_naive.py>
    <Function test_analytic_exact_shapley[12-combinatorial_exact_shapley-0.01-1e-05]>
    <Function test_analytic_exact_shapley[6-permutation_exact_shapley-0.01-1e-05]>
    <Function test_linear[2-0-20-r2]>
    <Function test_linear[2-1-20-r2]>
    <Function test_linear[2-1-20-neg_median_absolute_error]>
    <Function test_linear[2-1-20-explained_variance]>
    <Function test_grouped_linear[2-0-50-3-r2]>
    <Function test_grouped_linear[2-1-100-5-r2]>
    <Function test_grouped_linear[2-1-100-5-explained_variance]>
    <Function test_linear_with_outlier[2-1-20-explained_variance]>
    <Function test_linear_with_outlier[2-0-20-r2]>
    <Function test_linear_with_outlier[2-1-20-neg_median_absolute_error]>
    <Function test_linear_with_outlier[2-1-20-r2]>
    <Function test_polynomial[coefficients0-r2]>
    <Function test_polynomial[coefficients1-neg_median_absolute_error]>
    <Function test_polynomial[coefficients2-explained_variance]>
    <Function test_polynomial_with_outlier[coefficients0-r2]>
    <Function test_polynomial_with_outlier[coefficients1-neg_median_absolute_error]>
    <Function test_polynomial_with_outlier[coefficients2-explained_variance]>
  <Module test_truncated.py>
    <Function test_tmcs_analytic_montecarlo_shapley[joblib-12-truncated_montecarlo-0.1-1e-05-kwargs0]>
    <Function test_tmcs_analytic_montecarlo_shapley[ray-local-12-truncated_montecarlo-0.1-1e-05-kwargs0]>
    <Function test_tmcs_analytic_montecarlo_shapley[ray-external-12-truncated_montecarlo-0.1-1e-05-kwargs0]>
    <Function test_tmcs_linear_montecarlo_shapley[truncated_montecarlo-kwargs0-scorer0-0.25-2-0-21]>
    <Function test_tmcs_linear_montecarlo_with_outlier[truncated_montecarlo-kwargs0-scorer0-0.2-2-0-21]>

```

It only collects tests from the tests directory.
The reason why I'm insisting on this is that I sometimes select specific directories or test modules to run. For example, to run only the tests in test_parallel.py I would use:

```shell
tox -e base -- tests/utils/test_parallel.py
```
Your change would prevent me from doing this.

Reply:
Done